link_shortener/form_routes.py

```diff
@@ -26,14 +26,65 @@
 class CreateForm(SanicForm):
     endpoint = StringField('Endpoint', validators=[DataRequired()])
     url = StringField('URL', validators=[DataRequired()])
+    password = StringField('Password', validators=[DataRequired()])
```

Comment:
Is it intentional that you have not used the `PasswordField`?

Reply:
Fixed